.scripts/bump-version/maven/xml_maven_project.py

```diff
@@ -57,7 +58,7 @@ def bump_version(
                 )
 
         updated_versions: dict[str, str] = self._bump_versions(
-            current_versions, bump_type
+            current_versions, bump_type, custom_version
```

Comment:
(Minor)

Not ideal argument namings with combined `current_versions: dict[str,str]` and `current_version [str]`, requires the reader/maintainer to look it up first.